pkg/collector/corechecks/snmp/test/conf.d/snmp.d/profiles/_generic-if.yaml

```diff
@@ -59,3 +59,11 @@ metrics:
       name: ifAlias
     table: ifXTable
     tag: interface_alias
+  - column:
```

Comment:
What should we do for metrics knowing that we might have multiple IP on the same interface, multiple tags `ip_address` ?